Doc/library/concurrent.futures.rst

```diff
@@ -82,6 +82,11 @@ Executor Objects
        value of *wait*, the entire Python program will not exit until all
        pending futures are done executing.
 
+       If *cancel_futures* is ``True``, this method will cancel all pending
+       futures that the executor has not started running. Any futures that
+       are completed or running won't be cancelled, regardless of the value
+       of *cancel_futures*.
```

Comment:
I think that you might need to explain what happens when `cancel_futures` And `wait` are both true? Does `wait` cause `shutdown` to not return until all of the currently running futures are finished?

Reply:
Done.